src/Enterspeed.Source.UmbracoCms/Enterspeed.Source.UmbracoCms.csproj

```diff
@@ -24,7 +24,6 @@
 
 	<ItemGroup Condition="'$(TargetFramework)' == 'net5.0'">
 		<PackageReference Include="Umbraco.Cms.Core" Version="9.4.0" />
-
 		<PackageReference Include="Umbraco.Cms.Web.Website" Version="9.4.0" />
 		<PackageReference Include="Umbraco.Cms.Web.BackOffice" Version="9.4.0" />
 	</ItemGroup>
```

Comment:
Why do we want that?
Didn't we agree that we ignored the fact that the type might not exist, since it would never execute the code with the new grid component? 

@jesperweber

Reply:
Yes. Since we cant compile the project if we dont, due to the new BlogGridModel type, correct?

Reply:
correct